packages/ui/src/2_molecules/TableBase/TableBase.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { ReactNode } from 'react';
+
+import classNames from 'classnames';
+
+import styles from './TableBase.module.css';
+import { ColumnOptions, RowObject } from './TableBase.types';
+import { TableRow } from './components/TableRow/TableRow';
+import rowStyles from './components/TableRow/TableRow.module.css';
+
+export type TableBaseProps<RowType extends RowObject> = {
+  className?: string;
+  columns: ColumnOptions<RowType>[];
+  rows?: RowType[];
+  rowKey?: (row: RowType) => number | string;
+  noData?: ReactNode;
+  onRowClick?: (row: RowType) => void;
+  dataAttribute?: string;
```

Comment:
any reason you've changed ``dataLayoutId`` to this?

Reply:
yes, because we discussed it in https://sovryn.atlassian.net/browse/SOV-861 so I wanted to save soulBit some time :D